app/src/main/java/com/main/mealplanner/MainActivity.kt

```diff
@@ -10,7 +10,6 @@ class MainActivity : AppCompatActivity() {
     private lateinit var mainFragment: MainFragment
 
     override fun onCreate(savedInstanceState: Bundle?) {
-        mainFragment = MainFragment.newInstance()
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
         mainFragment = MainFragment.newInstance()
```

Comment:
There are two same lines, removed one of them.
Source:
https://kotlinlang.org/docs/coding-conventions.html#function-names

Reply:
Good. 

This did come up in a previous review as well.